server.js

```diff
@@ -6,12 +6,24 @@ config();
 
 const uriDb = process.env.DB_HOST;
 
+const startAnimation = () => {
+  const animation = ['| ', '/ ', '- ', '\\ '];
+  let index = 0;
+
+  setInterval(() => {
+    process.stdout.write('\r' + animation[index]);
+    index = (index + 1) % animation.length;
+  }, 80);
+};
+
 const runServer = async () => {
   try {
     const connection = await mongoose.connect(uriDb);
     console.log('Database connection successful');
     app.listen(3000, () => {
-      console.log('Server running. Use our API on port: 3000');
+      console.log('Host: 127.0.0.1:3000/api/contacts/');
```

Comment:
To nie prawda. Ten wcześniejszy log który usunąłeś był bardziej poprawny ;)

Reply:
Przywrócę go Asap jutro 😜